test/test_external.py

```diff
@@ -329,7 +329,7 @@ def test_loading_files_via_proxy_works(self):
         app, _, _ = io.launch(prevent_thread_lock=True)
         test_client = TestClient(app)
         r = test_client.get(
-            "/proxy=https://gradio-tests-test-loading-examples-private.hf.space/file=/tmp/tmprahzj703/Bunnyual53t2x.obj"
+            "/proxy=https://gradio-tests-test-loading-examples-private.hf.space/file=/tmp/tmpdd31pyri/Bunnyp0c5zw__.obj"
```

Comment:
Unrelated test that needed to be fixed because the Space went to sleep. I've upgraded the Space so that it should stay running and this test should pass in the future.